app/Http/Controllers/Api/AliasController.php

```diff
@@ -80,32 +86,42 @@ public function store(StoreAliasRequest $request)
                 'extension' => $extension ?? null
             ];
         } else {
+            $prefix = null;
+            if (isset($request->validated()['hostname'])) {
+                // TODO this should be cached, perhaps on boot, from https://publicsuffix.org/
+                // see https://github.com/jeremykendall/php-domain-parser
+                $publicSuffixList = Rules::fromPath(sys_get_temp_dir() . '/public_suffix_list.dat');
```

Comment:
Need to work out a way to do this, wasn't sure if there was anything else that might be cached in a similar way?

(Also doesn't seem like PHP has an equivalent to [path.Join](https://pkg.go.dev/path#Join) so hopefully this is platform agnostic.)